README.md

```diff
@@ -486,43 +486,84 @@ https://github.com/dwyl/app/issues/234
-      ```elixir
-      :crypto.strong_rand_bytes(32) |> :base64.encode
-      ```
-    - ALLOW_API_ORIGINS: origins allowed to request the application API
-
-
-  * run `source .env` to load the enviroment variables
-  * Make sure you have Postgres installed, see: https://github.com/dwyl/learn-postgresql
-  * Install dependencies with `mix deps.get`
-  * Create and migrate your database with `mix ecto.setup`
-  * Install Node.js dependencies with `cd assets && npm install`
-  * Start Phoenix endpoint with `mix phx.server`
-
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
-
-Ready to run in production? Please
-[check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
-
-
-## Learn more
-
-  * Official website: http://www.phoenixframework.org/
-  * Guides: https://hexdocs.pm/phoenix/overview.html
-  * Docs: https://hexdocs.pm/phoenix
-  * Mailing list: http://groups.google.com/group/phoenix-talk
-  * Source: https://github.com/phoenixframework/phoenix
+If you want to _run_ this App on your `localhost`,
+please _first_ ensure that you have PostgreSQL installed.
+see:
+[Prerequisites](https://github.com/dwyl/phoenix-chat-example#0-pre-requisites-before-you-start)
+
+### 1. `git clone`
+
+Clone the repository from GitHub:
+
+```
+git clone git@github.com:dwyl/app-mvp-phoenix.git && cd app-mvp-phoenix
+```
+
+### 2. Required Environment Variables
+
+Create an `.env` file to store the required environment variables:
+```sh
+cp .env_sample .env
+```
+
+The
+[`.env_sample`](https://github.com/dwyl/app-mvp-phoenix/blob/master/.env_sample)
+file shows which environment variables are required.
+
+For running on `localhost`, you will only need to change the values
+for `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` to _real_ values
+in order for Google Auth to work.
+For that you will need to follow the instructions:
+[create-google-app-guide.md](https://github.com/dwyl/elixir-auth-google/blob/master/create-google-app-guide.md)
+
+> If you are new to Environment Variables, please see:
+[dwyl/**learn-environment-variables**](https://github.com/dwyl/learn-environment-variables)
+
+Once you have updated the `.env` file with the
+`GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` API keys
+you have everything you need to run the app.
+
+Run `source .env` in your terminal
+to load the environment variables.
+
+### 3. Install Dependencies
+
+Install the `Elixir` dependencies by running
+
+```
+mix deps.get
+```
+
+Install Node.js dependencies by running:
+
+```
+cd assets && npm install && cd ..
+```
+
+
+### 4. Create the Database
+
+Create and migrate your database with the command:
+```
+mix ecto.setup
+```
+That should create a new database called `app_dev`.
+
+
+### Start the App
+
+Finally once everything is installed and the DB is created,
+run the app:
+
+```
+mix phx.server
+```
+
+Visit [`localhost:4000`](http://localhost:4000) in your web browser.
+You should see somethign similar to the following:
```

Comment:
```suggestion
You should see something similar to the following:
```

Reply:
Tried to commit the suggested change but got a "Sorry ..." error message:
![image](https://user-images.githubusercontent.com/194400/73288550-185fa080-41f3-11ea-8d9d-693051790e7e.png)
If you are going to make a suggestion, please don't then override it with your own commit. 👍